portfolio/src/main/webapp/script.js

```diff
@@ -15,13 +15,18 @@
 /**
  * Show/hide the description inside the element which has title as id.
  */
-function toggleContent(title) {
-  var descr = document.getElementById(title).getElementsByClassName("description");
-  if (descr && descr.length>0) {
-    if (descr[0].style.display === "none") {
-      descr[0].style.display = "block";
+function toggleContent(sectionId) {
+  if (document.getElementById(sectionId) == null) {
```

Comment:
nice! I think this is a really readable exit condition. You can even shorten it further and just write
`if(!document.getElementById(sectionId))`